### DISCUSSION TITLE
NewMediaGroup , cannot send list video when use  []tgbotapi.InputMediaVideo

### DISCUSSION THREAD

**Discussion by radiaku** [go-telegram-bot-api/telegram-bot-api#428] [Q&A] [answered]
Hello I got error when send multiple file using []tgbotapi.InputMediaVideo as interface in newmediagroup

my code like this...

```
var listMediaVideoInput []tgbotapi.InputMediaVideo
for _, value := range dataListMediaVideo {
	var tempVideoInput = tgbotapi.NewInputMediaVideo(value)
	listMediaVideoInput = append(listMediaVideoInput, tempVideoInput)
}
msg = tgbotapi.NewMediaGroup(update.Message.Chat.ID, listMediaVideoInput)
bot.Send(msg)
```

I got error with this

> .\main.go:226:9: cannot use tgbotapi.NewMediaGroup(update.Message.Chat.ID, listMediaVideoInput) (type tgbotapi.MediaGroupConfig) as type tgbotapi.MessageConfig in assignment
> .\main.go:226:53: cannot use listMediaVideoInput (type []tgbotapi.InputMediaVideo) as type []interface {} in argument to tgbotapi.NewMediaGroup

Did I do something wrong?

**Comment by Syfaro** [comment, marked as answer]
You need to use `[]interface{}` instead of `[]tgbotapi.InputMediaVideo` because that's the type `NewMediaGroup` is expecting: https://github.com/go-telegram-bot-api/telegram-bot-api/blob/54104a08f947e08c460c45d195076e48e61f0a26/helpers.go#L320

```go
var listMediaVideoInput []interface{}
for _, value := range dataListMediaVideo {
	var tempVideoInput = tgbotapi.NewInputMediaVideo(value)
	listMediaVideoInput = append(listMediaVideoInput, tempVideoInput)
}
msg = tgbotapi.NewMediaGroup(update.Message.Chat.ID, listMediaVideoInput)
bot.Send(msg)
```
